gems/line-bot-api/1.25/manifest.yaml

```diff
@@ -0,0 +1,4 @@
+dependencies:
```

Comment:
Could you remove `uri` and `timeout` from this file? `rbs collection` loads `manifest.yaml` recursively and net-http's `manifest.yaml` contains them.
https://github.com/ruby/rbs/blob/39f95989418df9ca3938f7f81df5de71db07ce7f/stdlib/net-http/0/manifest.yaml#L1-L3
I haven't tested removing them, so feel free to ask me if you have any trouble removing them.

---


By the way, `Steepfile` needs them because unfortunately Steep doesn't use `rbs collection` feature yet in this repository. I'm trying to enable this feature. Sorry for the inconvenience.

Reply:
I was confusing RBS and Steep dependencies.
I removed them because the steep check passed without any problems.

Thank you for efforts to improve Steep.

Reply:
fix:
https://github.com/ruby/gem_rbs_collection/pull/206/commits/7d5468276efefae0367cdff96a15863adc1ceac7